lab-kyle/index.js

```diff
@@ -0,0 +1,5 @@
+'use strict';
+
+const read = require('./lib/read.js');
+
+read.readFiles(['./texts/one.txt', './texts/two.txt', './texts/three.txt'], read.returnShit);
```

Comment:
I'll give you this one for free... name things appropriately though going forward.